src/Stratis.Bitcoin.Features.Consensus/CoinViews/CachedCoinView.cs

```diff
@@ -59,7 +59,7 @@ private class CacheItem
 
         /// <summary>Length of the coinview cache flushing interval in seconds.</summary>
         /// <seealso cref="lastCacheFlushTime"/>
-        public const int CacheFlushTimeIntervalSeconds = 60;
+        public const int CacheFlushTimeIntervalSeconds = 600;
```

Comment:
If your increasing the flush time X 10 - maybe change the constant to CacheFlushTimeIntervalInMinutes =  10;

Unless it may come back down to seconds in the future.